interdiff.go

```diff
@@ -236,70 +284,81 @@ func compareOverlappedHunks(oldHunks, newHunks []*diff.Hunk) *diff.Hunk {
 
 	for _, line := range newBody {
 		if !strings.HasPrefix(line, " ") {
-			return resultHunk
+			// resultHunkBody contains some changes
+			return resultHunk, nil
 		}
 	}
 
-	return nil
+	return nil, nil
 }
 
-func configureResultHunk(oldHunks, newHunks []*diff.Hunk) (*diff.Hunk, int32) {
+func configureResultHunk(oldHunks, newHunks []*diff.Hunk) (*diff.Hunk, int32, error) {
 	if (len(oldHunks) == 0) || (len(newHunks) == 0) {
-		return nil, 0
+		return nil, 0, fmt.Errorf("one of the hunks array is empty")
```

Comment:
```suggestion
		return nil, 0, errors.New("one of the hunks array is empty")
```

If you're not using any format strings, it's recommended to use `errors.New()` which just takes a regular string value. You'll also need to add that new import.